language/compiler/src/main.rs

```diff
@@ -73,10 +72,13 @@ fn write_output(path: &PathBuf, buf: &[u8]) {
 fn main() {
     let args = Args::from_args();
 
-    let address = args
-        .address
-        .map(|a| AccountAddress::try_from(a).unwrap())
-        .unwrap_or(account_config::CORE_CODE_ADDRESS);
+    let address = match AccountAddress::from_hex_literal(&args.address) {
+        Ok(address) => address,
+        Err(_) => {
+            println!("Bad address");
```

Comment:
Print the error and/or malformed address?

Reply:
Yeah, I'd probably just spit out "Malformed address: {}"

Doesn't need to be super detailed for the IR